app/models/listing.rb

```diff
@@ -1,4 +1,4 @@
 class Listing < ApplicationRecord
-  validates_presence_of :category, :lead, :scene
+  validates_presence_of :category, :lead, :scene, :height, :description, :price
```

Comment:
Should be "dimensions" instead of just "height". You might have different kind of spaces were the dimensions would be important